questions/javascript-questions.md

```diff
@@ -64,3 +64,7 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are the benefits of using `spread syntax` and how is it different from `rest syntax`?
 * How can you share code between files?
 * Why you might want to create static class members?
+* If you have a function that generate random number between 1 to 5 how could u generate random number 1 to 7 by using that function?
```

Comment:
don't think it's related to javascript

Reply:
Yes, it is a javacript question. Can be solved by using Math.random().

Thanks.

Reply:
It is not `javascript` question, just because it's language agnostic. You don't ask implement function which returns random between 1 and 5, you say that the function is already exists

Reply:
Ok. Understood.

I will add some new questions.